contracts/payloads/IGP80/PayloadIGP80.sol

```diff
@@ -0,0 +1,583 @@
+    function action5() internal {
+        address USDe_USDT_DEX_ADDRESS = getDexAddress(18);
+        address USDC_USDT_DEX_ADDRESS = getDexAddress(2);
+        address USDe_USDT__USDC_USDT_VAULT_ADDRESS = getVaultAddress(99);
+
+        {
+            // Update USDe-USDT<>USDC-USDT vault supply shares limit
+            IFluidAdminDex.UserSupplyConfig[]
+                memory config_ = new IFluidAdminDex.UserSupplyConfig[](1);
+            config_[0] = IFluidAdminDex.UserSupplyConfig({
+                user: USDe_USDT__USDC_USDT_VAULT_ADDRESS,
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours, // 12 hours
+                baseWithdrawalLimit: 10_000 * 1e18 // 10k shares
+            });
+
+            IFluidDex(USDe_USDT_DEX_ADDRESS).updateUserSupplyConfigs(config_);
+        }
+
+        {
+            // Update USDe-USDT<>USDC-USDT vault borrow shares limit
+            IFluidAdminDex.UserBorrowConfig[]
+                memory config_ = new IFluidAdminDex.UserBorrowConfig[](1);
+            config_[0] = IFluidAdminDex.UserBorrowConfig({
+                user: USDe_USDT__USDC_USDT_VAULT_ADDRESS,
+                expandPercent: 20 * 1e2, // 20%
+                expandDuration: 12 hours, // 12 hours
+                baseDebtCeiling: 10_000 * 1e18, // 10k shares
+                maxDebtCeiling: 20_000 * 1e18 // 20k shares
+            });
+
+            IFluidDex(USDC_USDT_DEX_ADDRESS).updateUserBorrowConfigs(config_);
+        }
+
+        VAULT_FACTORY.setVaultAuth(
+                USDe_USDT__USDC_USDT_VAULT_ADDRESS,
+                TEAM_MULTISIG,
+                true
+            );
+    }
+
+    // @notice Action 6: Update wstETH-ETH DEX Lower Range
+    function action6() internal {
+        address wstETH_ETH_DEX_ADDRESS = getDexAddress(1);
+
+        {
+            // Update Lower Range
+            IFluidDex(wstETH_ETH_DEX_ADDRESS).updateRangePercents(
+                0.0001 * 1e4,
+                0.075 * 1e4,
+                0
+            );
+        }
+    }
+
+    // @notice Action 7: Update weETHs-ETH DEX Lower Range and Fee %
+    function action7() internal {
+        address weETHs_ETH_DEX_ADDRESS = getDexAddress(14);
+
+        {
+            // Update Lower Range
+            IFluidDex(weETHs_ETH_DEX_ADDRESS).updateRangePercents(
+                0.0001 * 1e4,
+                0.1 * 1e4,
+                5 days
+            );
+        }
+        
+        {
+            //Update Trading Fee
+            IFluidDex(weETHs_ETH_DEX_ADDRESS).updateFeeAndRevenueCut(
+                0.05 * 1e4,
+                0
+            );
+        }
+    }
+
+    // @notice Action 8: Withdraw ETH from Reserve to team multisig
+    function action8() internal {
+        address[] memory tokens = new address[](1);
+        uint256[] memory amounts = new uint256[](1);
+
+        tokens[0] = ETH_ADDRESS;
+        amounts[0] = address(FLUID_RESERVE).balance;
```

Comment:
this would withdraw all 1545 ETH, that is too much 

we should withdraw 500 ETH, this will fund rewards for ~1 month + depending on ETH price